auraed/src/runtime/cell_service/executables/executables.rs

```diff
@@ -90,9 +91,9 @@ impl Executables {
 
         let Some(exit_status) = exit_status else {
             // Exes that never started return None
-            let executable = self.cache.remove(executable_name).expect("exe in cache");
+            let _ = self.cache.remove(executable_name).expect("exe in cache");
             return Err(ExecutablesError::ExecutableNotFound {
-                executable_name: executable.name,
+                executable_name: executable_name.clone(),
```

Comment:
Why introduce a clone here? We know that the exe is in the cache (hence the use of `expect("exe in cache")`) since we didn't error in `self.get_mut` which is the first call in `stop`.

Reply:
otherwise it would be a move from a shared reference.  and i can't get the name from the executable anymore without a lock, which i'd like to avoid in this flow.